README.md

```diff
@@ -22,24 +22,24 @@ $ yarn add -D kremling-loader
 
 ```js
 // webpack.config.js
-...
-config.module = {
-  rules: [
-    {
-      test: /\.css$/,
-      use: [
-        {
-          loader: 'kremling-loader',
-          options: {
-            namespace: 'custom-namespace',
-            postcss: {},
+module.exports = {
```

Comment:
most people have their webpack config like this, instead of reassigning `config.module` after already writing it.

Reply:
^ agreed